webtool/frontend/src/index.css

```
@@ -246,8 +246,67 @@ a.blatt:hover, .blatt-klickbar:hover {
+     `@layer components`). Dieselbe Lehre wie #209, zwoelf Zeilen weiter in `MaterialDialog`.
+   - **Der Radius ist um die 2 px groesser als der des Dialogs** — `border-radius: inherit`
+     ergaebe aussen dieselbe Rundung wie innen, und die Ecke liefe sichtbar spitz zu. */
+.rahmen-animiert::before {
+  content: "";
+  position: absolute;
+  inset: -2px;
+  padding: 2px;
+  border-radius: calc(var(--radius) + 2px);
+  --rahmen-winkel: 0deg;
+  background: conic-gradient(from var(--rahmen-winkel),
+              var(--primary), #22D3EE, #A78BFA, var(--primary));
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)